### PR TITLE
Changes fireman carrying from a trait on gloves to a skillchip

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

What it says on the tin. 

Latex gloves were readily available in significant numbers. So I put a box of them in the medical lockers. Latex gloves are cheap and plentiful, so the skillchips here should be as well. Every medical doctor starts with this chip implanted.

However, nitrile gloves are special. There aren't as many available in a round, with one in each medical locker, one in the CMO's, and spawned with paramedics. Therefore, the CMO gets a box of the skillchips in their locker. They and paramedics also start with one implanted.

## Why It's Good For The Game

This fits a skillchip way more than a glove trait, since it is literally a skill imparted by the gloves.

## Changelog
:cl:
balance: Changes the carrying traits on Latex/Nitrile gloves into a skillchip.
balance: Medical doctors start the round with a quick carry skillchip (was the latex glove trait).
balance: Paramedics and the CMO start the round with a quicker carry skillchip (was the nitrile glove trait).
/:cl:

### CODE REVIEW

**Comment by coiax** [comment]
Literally was thinking about doing this the other day, a good candiate for skillchips.

File: code/modules/library/skill_learning/skillchip.dm [hunk 1]
```diff
@@ -433,3 +433,27 @@
 	skill_icon = "lungs"
 	activate_message = "<span class='notice'>You feel that you know a lot about interpreting organs.</span>"
 	deactivate_message = "<span class='notice'>Knowledge of liver damage, heart strain and lung scars fades from your mind.</span>"
+
+/obj/item/skillchip/quickcarry
+	name = "Ant Hauler skillchip"
+	auto_traits = list(TRAIT_QUICK_CARRY)
+	skill_name = "Ant Hauler"
+	chip_category = "bodycarrying"
```
Comment (coiax) [inline code/modules/library/skill_learning/skillchip.dm:441]: Should this category be a define?

**Comment by Fikou** [comment]
tinkerer gloves dont have a reason to be latex subtypes anymore

File: code/modules/library/skill_learning/skillchip.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/library/skill_learning/skillchip.dm:441]: Yeah probably.

**Comment by necromanceranne** [comment]
> tinkerer gloves dont have a reason to be latex subtypes anymore

Why were they before?

File: code/modules/library/skill_learning/skillchip.dm (hunk 1 above)
Comment (Jared-Fogle) [reply code/modules/library/skill_learning/skillchip.dm:441]: yeah

**Review by Mothblocks — CHANGES_REQUESTED** [review]
make that thing a define

personally as a medical doctor i don't like that the slight nerf on me so much but it makes sense so i bite my tongue

**Comment by coiax** [comment]
> 
> 
> make that thing a define
> 
> personally as a medical doctor i don't like that the slight nerf on me so much but it makes sense so i bite my tongue

Just drag a braindead paramedic to the library, and steal his chip.

**Comment by necromanceranne** [comment]
Did some categories defines hope that's enough

**Comment by Jared-Fogle** [comment]
> Just drag a braindead paramedic to the library, and steal his chip. 🔪 🧠

i just get promoted to cmo every round anyway i'll just steal it from the locker so much work though

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-10). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**